frontend/public/components/dashboard/dashboards-page/cluster-dashboard/utilization-card.tsx

```diff
@@ -175,12 +176,26 @@ export const PrometheusUtilizationItem = withDashboardResources<PrometheusUtiliz
     let request: PrometheusResponse, requestError: any;
     let isLoading = false;
 
+    const { t } = useTranslation();
+
+    const UTILIZATION_QUERY_DURATION = {
+      ONE_HR: t('public~1 hour'),
```

Comment:
why do we need these string both to be in `public` and `console-shared` locales? Is it OK if they are duplicate ?

Reply:
It shouldn't cause problems, but I can consolidate them into one namespace.